src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -0,0 +1,2583 @@
+  "modifier.tconstruct.blaze_disguise": "烈焰伪装",
+  "modifier.tconstruct.blaze_disguise.flavor": "如果你着火了会更有说服力",
+  "modifier.tconstruct.blaze_disguise.description": "让你在其他火焰中看起来更像火焰",
+  "modifier.tconstruct.piglin_disguise": "猪灵伪装",
+  "modifier.tconstruct.piglin_disguise.flavor": "需要一些金币来完成伪装",
+  "modifier.tconstruct.piglin_disguise.description": "让你看起来更像其他猪灵的猪灵",
+  "modifier.tconstruct.piglin_brute_disguise": "猪灵蛮兵伪装",
+  "modifier.tconstruct.piglin_brute_disguise.flavor": "这个伤疤有说服力吗？",
+  "modifier.tconstruct.piglin_brute_disguise.description": "让你在其他猪灵蛮兵眼中更像猪灵蛮兵",
+  "modifier.tconstruct.zombified_piglin_disguise": "僵尸猪灵伪装",
+  "modifier.tconstruct.zombified_piglin_disguise.flavor": "是猪？人？僵尸？",
+  "modifier.tconstruct.zombified_piglin_disguise.description": "让你在其他僵尸猪灵看来更像僵尸猪灵",
+
+  "_comment": "Compat Modifiers",
+  "modifier.tconstruct.the_one_probe": "一个探测针",
+  "modifier.tconstruct.the_one_probe.flavor": "科学！",
+  "modifier.tconstruct.the_one_probe.description": "当持有或戴在头盔上时显示目标方块的信息。",
+
+  "_comment": "Internal effects",
+  "effect.tconstruct.bleeding": "流血",
+  "effect.tconstruct.magnetic": "磁性",
+  "effect.tconstruct.momentum": "动量",
+  "effect.tconstruct.insatiable": "贪婪",
+  "effect.tconstruct.teleport_cooldown": "传送冷却时间",
+  "effect.tconstruct.fireball_cooldown": "火球冷却时间",
+  "effect.tconstruct.calcified": "钙化",
+
+  "_comment": "Roman numerals",
+  "roman_numeral.value.1": "I",
+  "roman_numeral.value.2": "II",
+  "roman_numeral.value.3": "III",
+  "roman_numeral.value.4": "IV",
+  "roman_numeral.value.5": "V",
+  "roman_numeral.value.6": "VI",
+  "roman_numeral.value.7": "VII",
+  "roman_numeral.value.8": "VIII",
+  "roman_numeral.value.9": "IX",
+  "roman_numeral.value.10": "X",
+  "roman_numeral.value.11": "XI",
+  "roman_numeral.value.12": "XII",
+  "roman_numeral.value.13": "XIII",
+  "roman_numeral.value.14": "XIV",
+  "roman_numeral.value.15": "XV",
+  "roman_numeral.value.16": "XVI",
+  "roman_numeral.value.17": "XVII",
+  "roman_numeral.value.18": "XVIII",
+  "roman_numeral.value.19": "XIX",
+  "roman_numeral.value.20": "XX",
+
+  "_comment": "Stats",
+
+  "tool_stat.tconstruct.attack_speed": "攻击速度:",
+  "tool_stat.tconstruct.attack_speed.description": "计算攻击速度的基础值。\n大约是每秒全强度攻击。",
+  "tool_stat.tconstruct.durability": "耐用性:",
+  "tool_stat.tconstruct.durability.description": "耐用性计算的基础值。\n通常是工具整体耐用性的组成部分。",
+
+  "stat.tconstruct.head": "头部",
+  "tool_stat.tconstruct.mining_speed": "挖掘速度:",
+  "tool_stat.tconstruct.mining_speed.description": "带有这种材料刀头的工具开采方块的速度有多快。\n可能会受到其他工具部件的影响。",
+  "tool_stat.tconstruct.harvest_tier": "挖掘等级:",
+  "tool_stat.tconstruct.harvest_tier.description": "带有这种材料工具头的工具可以开采哪些的方块。",
+  "tool_stat.tconstruct.attack_damage": "攻击伤害:",
+  "tool_stat.tconstruct.attack_damage.description": "攻击计算的基础值。\n最终结果取决于工具本身和使用的其他组件。",
+
+  "tool_stat.tconstruct.armor": "护甲:",
+  "tool_stat.tconstruct.armor.description": "这对你有多大的保护作用。",
+  "tool_stat.tconstruct.armor_toughness": "装甲韧性:",
```

```suggestion
  "tool_stat.tconstruct.armor_toughness": "盔甲韧性：",
```
原版专有名词，应保持一致。
所有类似情况可参考中文Minecraft Wiki、进入游戏或查看原版的zh_cn.json以确定原版译文。